source/_docs/configuration/templating.markdown

```diff
@@ -357,6 +357,8 @@ To fix it, enforce the ISO conversion via `isoformat()`:
 
 The `to_json` filter serializes an object to a JSON string. In some cases, it may be necessary to format a JSON string for use with a webhook, as a parameter for command-line utilities or any number of other applications. This can be complicated in a template, especially when dealing with escaping special characters. Using the `to_json` filter, this is handled automatically.
 
+Similarly to the python equivalent, the filter accepts a `ensure_ascii` parameter, defaulting to `True`, which specifies the encoding of the resulting JSON: UTF-8 or escaped ASCII.  
```

Comment:
```suggestion
Similarly to the python equivalent, the filter accepts an `ensure_ascii` parameter, defaulting to `True`. If `ensure_ascii` is `True`, the output is guaranteed to have all incoming non-ASCII characters escaped. If `ensure_ascii` is false, these characters will be output as-is.
```

Reply:
"as-is" does not mean much in technical terms ;)
But if you think it's clearer to users, fine by me.

Reply:
The suggestion is copied verbatim from the official Python documentation.
If you don't like "as-is", "unchanged" would be fine too.

Reply:
Nah, as-is is clear enough for non-techies